src/bucket/BucketController.ts

```diff
@@ -113,6 +109,12 @@ export class BucketController {
    * Initialize the controller.
    */
   $onInit() {
+    // Set the currentFolder based on bucket prefix
+    if(this.awsBucketPrefix === undefined){
```

Comment:
I think you'll see when you rebase that there are two whitespace style errors that should be fixed here.